api/src/pcapi/core/finance/api.py

```diff
@@ -1416,9 +1420,10 @@ def _mark_as_processed(pricings: sqla_orm.Query) -> None:
     reimbursement_point_infos = (
         models.Pricing.query.filter(*filters)
         .outerjoin(models.Pricing.booking)
-        .outerjoin(models.Pricing.collectiveBooking)
         .outerjoin(bookings_models.Booking.stock)
+        .outerjoin(models.Pricing.collectiveBooking)
         .outerjoin(educational_models.CollectiveBooking.collectiveStock)
+        .outerjoin(models.Pricing.event)
```

Comment:
[détail] Il y aura bientôt une contrainte `NOT NULL` sur `Pricing.eventId` et, en pratique, cette colonne est déjà systématiquement remplie. On peut donc faire une jointure simple : 

```suggestion
        .join(models.Pricing.event)
```